flow/project.py

```diff
@@ -3401,7 +3887,8 @@ def _register_groups(self):
                 operation_directives = getattr(func, '_flow_group_operation_directives', dict())
                 for group_name in func._flow_groups:
                     self._groups[group_name].add_operation(
-                        op_name, op, operation_directives.get(group_name, None))
+                        op_name, op, operation_directives.get(group_name, None),
+                        flow_aggregate[op_name], flow_select[op_name])
 
             # For singleton groups add directives
             self._groups[op_name].operation_directives[op_name] = getattr(func,
```

Comment:
We need to set the singleton groups aggregation and selection here.

Reply:
I'm not sure what do you mean by this. Can you please explain?